cartography/intel/gcp/crm.py

```diff
@@ -45,14 +49,119 @@ def load_gcp_organizations(neo4j_session, data, gcp_update_tag):
         )
 
 
+def load_gcp_folders(neo4j_session, data, gcp_update_tag):
```

Comment:
I'm going to be annoying and request that you separate data transformation from loading in this and the below function. That is:
```
load_gcp_folders(session, folders, gcp_update_tag)
```
should become:
```
transformed_data = transform_gcp_folders(folders)
load_gcp_folders(session, transformed, gcp_update_tag)
```
and the same for the function below.